easypermissions/src/main/java/pub/devrel/easypermissions/AppSettingsDialog.java

```diff
@@ -66,14 +68,16 @@ private AppSettingsDialog(@NonNull final Object activityOrFragment,
                               @Nullable String title,
                               @Nullable String positiveButtonText,
                               @Nullable String negativeButtonText,
-                              int requestCode) {
+                              int requestCode,
+                              boolean openInNewTask) {
```

Comment:
Even though the `Builder` method is `setOpenInNewTask(boolean)` I think we should make the `AppSettingsDialog` object take an `int` here called `intentFlags` rather than a `boolean`?  What do you think?  IMO it makes the purpose clearer and lets us add more flags in the future without adding any new fields.

The default can be `0`.

Reply:
Yes, I thought about that option, too. I think it is a good idea